File: hw/Vaca/u2/HW18ZooInPythonJavaScript/Zoo/ec.edu.espe.zoo.model/Eagle.js
const Bird = require('./Bird');

class Eagle extends Bird {
    constructor(id, description, dateOnborn, gender, numberOfBones, cage, wingSpan) {
        super(id, description, dateOnborn, gender, numberOfBones, cage, wingSpan);
    }

    toString() {
        return `Eagle{wingSpan=${this.wingSpan}${super.toString()}}`;
    }
}

module.exports = Eagle;

File: hw/Rodriguez/u2/hw18/Zoo/ec_edu_espe_zoo_model/eagle.py
from .bird import Bird
from datetime import date
from .cage import Cage

class Eagle(Bird):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

    def feed(self, food):
        raise NotImplementedError("Subclasses should implement this method")
